### PR TITLE
unique merge array customisation fix

### LINKED ISSUE #161: mergeUnique array customization does not work properly
Hi,

The "unique" merge array customisation works only when the second array has the same element in the same place (has the same index).

Just exchange these two lines
https://github.com/survivejs/webpack-merge/blob/develop/test/unique.test.ts#L72,L73
which should not change the result and the test will fail with a duplicated plugin.

### PR DESCRIPTION
**Pull request by trombka**
This fixes #161

### CODE REVIEW

File: test/unique.test.ts [hunk 1]
```diff
@@ -69,8 +69,8 @@ describe("Unique", function () {
     })(
       {
         plugins: [
-          new webpack.HotModuleReplacementPlugin(),
           new webpack.DefinePlugin({}),
+          new webpack.HotModuleReplacementPlugin(),
```
Comment (bebraw) [inline test/unique.test.ts:73]: Can you duplicate the original test and do your change so we are sure there's no regression? I think after that this one is good to go. Thanks a lot!

Comment (trombka) [reply test/unique.test.ts:73]: Done. I have also added a note to the documentation on the ordering of elements after a merge.

Comment (bebraw) [reply test/unique.test.ts:73]: Awesome. I'll review, merge, and publish sometime today. 👍

**Comment by bebraw** [comment]
The fix is included in `5.4.1`. Thanks a lot for contributing!

**Pull request merged**